workers/environment.yml

```diff
@@ -11,9 +11,11 @@ dependencies:
   - greenelab::all-entrezg-brainarray-cdfs
   - python=3.6
   - r::rpy2=2.8.5
+  - pycurl=7.43.0
```

Comment:
You know what, this is a good question. I think I wanted to try it out for something but ended up not using it. I should remove this.